hugr-core/src/extension/prelude/array.rs

```diff
@@ -40,6 +40,7 @@ use crate::types::type_param::TypeParam;
 #[non_exhaustive]
 pub enum ArrayOpDef {
     new_array,
+    new_uninitialized_array,
     get,
```

Comment:
Move this to the end or add an out-of-order discriminant to avoid the breaking change.
```suggestion
    new_uninitialized_array=100,
    get=2,
```